content/en/docs/Installation/_index.md

```diff
@@ -644,3 +644,16 @@ kubectl delete mutatingwebhookconfigurations kyverno-policy-mutating-webhook-cfg
 
 kubectl delete validatingwebhookconfigurations kyverno-policy-validating-webhook-cfg kyverno-resource-validating-webhook-cfg
 ```
+## Recommended setup
+
+Kyverno recommended configurations on different scales of clusters
+
+### Kyverno Memory limit config:
+
+* less than 4000 total resource: use default limit
+* more than 4000 total resource: use 768Mi (2x the default)
```

Comment:
@husnialhamdani - were these data accurate? What was the memory limit during the test?

We may need to update these after the memory leak issue is fixed.